HISTORY.md

```diff
@@ -3,35 +3,41 @@
 
 This incorporates all changes after 1.3.5 up to 1.3.6.
 
+* deps: `path-to-regexp@6.2.0` (See [release changelog](https://github.com/pillarjs/path-to-regexp/releases/tag/v6.2.0))
+  - Support for **named capturing groups** in paths using `RegExp`, i.e.: `/\/(?<group>.+)/` would result in `req.params.group` being populated.
+  - Custom **prefix and suffix groups** using `{}`, e.g.: `/:entity{-:action}?` would match `/user` and `/user-delete`.
+  - **BREAKING**: Unbalanced patterns would now produce an error: `/test(foo` previously would worked, now it expects either `(` to be closed or the character to be escaped for the previous behavior, e.g. `/test\\(foo`.
+
 2.0.0-beta.1 / 2020-03-29
 =========================
 
-This incorporates all changes after 1.3.3 up to 1.3.5.
+This incorporates all changes after `1.3.3` up to `1.3.5`.
 
   * Internalize private `router.process_params` method
   * Remove `debug` dependency
-  * deps: array-flatten@3.0.0
-  * deps: parseurl@~1.3.3
-  * deps: path-to-regexp@3.2.0
-    - Add new `?`, `*`, and `+` parameter modifiers
+  * deps: `array-flatten@3.0.0`
+  * deps: `parseurl@~1.3.3`
+  * **BREAKING**:  deps: `path-to-regexp@3.2.0`
+    - Add new `?`, `*`, and `+` parameter modifiers.
     - Matching group expressions are only RegExp syntax.
       `(*)` is no longer valid and must be written as `(.*)`, for example.
     - Named matching groups no longer available by position in `req.params`.
       `/:foo(.*)` only captures as `req.params.foo` and not available as
-      `req.parmas[0]`.
+      `req.params[0]`.
     - Regular expressions can only be used in a matching group.
       `/\\d+` is no longer valid and must be written as `/(\\d+)`.
     - Special `*` path segment behavior removed.
       `/foo/*/bar` will match a literal `*` as the middle segment.
-  * deps: setprototypeof@1.2.0
+    - Parenthesis and bracket literals now require escaping, i.e.: `/user(:id)` or `/user{:id}` no longer match `/  user(42)`, `/user{42}` as before unless written as `/user\\(:id\\)` or `/user\\{:id\\}`.
```

Comment:
I think this got messed up in the move. From my recollection from the conversation only the parenthesis stuff changed in this release; the curly brace stuff did change in this PR. Only the parans one should have been moved down here, right?

Reply:
@dougwilson I think you're right. Corrected it. Also rebased with latest `2.0`.